packages/library-base/src/commonMain/kotlin/io/realm/kotlin/internal/RealmInstantImpl.kt

```diff
@@ -39,3 +40,6 @@ internal fun Duration.toRealmInstant(): RealmInstant {
     val nanos: Duration = (this - seconds.seconds)
     return RealmInstant.from(seconds, nanos.inWholeNanoseconds.toInt())
 }
+
+internal fun RealmInstant.withMillisPrecision() =
```

Comment:
Nitpick: Maybe rename to `RealmInstant.withMillisPrecision()` to make it clear that we are dropping  nanonseconds?

Reply:
🤔

Reply:
Ups, copy-past error. Fixed the name